gitCoreJGit/src/main/java/com/virtuslab/gitcore/impl/jgit/GitCoreLocalBranch.java

```diff
@@ -179,14 +181,17 @@ public String getBranchTypeString(boolean capitalized) {
 
     RevWalk walk = new RevWalk(repo.getJgitRepo());
     walk.sort(RevSort.TOPO);
-    RevCommit commit = resolveRevCommit(getPointedCommit().getHash().getHashString());
+    @Unique RevCommit commit = resolveRevCommit(getPointedCommit().getHash().getHashString());
     try {
       walk.markStart(commit);
     } catch (IOException e) {
       throw new GitCoreException(e);
     }
 
-    for (RevCommit currentBranchCommit : walk) {
+    // There's apparently no way for AliasingChecker to work correctly with generics
```

Comment:
See https://checkerframework.org/manual/#aliasing-unique-restrictions for the reason why